admin/app/components/solidus_admin/ui/pagination/component.html.erb

```diff
@@ -1,2 +1,33 @@
 <div>
+  <nav>
+    <ul class="flex items-center -space-x-px h-10 text-base">
```

Comment:
Is the `-` before `space-x-px` a typo? :thinking:

Reply:
Also, I wonder if `text-base` makes any difference here, as we only use `icon_tag` within the container. Is it because of the line-height?